client/lxd_certificates.go

```diff
@@ -20,7 +20,7 @@ func (r *ProtocolLXD) GetCertificateFingerprints() ([]string, error) {
 	}
 
 	// Parse it.
-	return urlsToResourceNames(baseURL, urls...)
+	return api.URLsToResourceNames(baseURL, urls...)
```

Comment:
@masnax given what we've discussed now, is it possible to remove this commit, as I don't want to move it to the `share/api` package only to remove it shortly after (as it is considered a public package that others may depend on).

Reply:
Sure. I'd have to duplicate (or make public, but that would have a larger diff) `urlsToResourceNames` in the db package temporarily.

Reply:
Yeah client is shared too, so making it public is no good. Happy for it to be duplicated if only for a short time.

Reply:
Tests seem to be passing so I think it's good to merge now.